dev-libs/libflatarray/libflatarray-0.2.0-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake cuda
+
+SRC_URI="http://www.libgeodecomp.org/archive/${P}.tar.bz2"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+DESCRIPTION="Struct of arrays library with object oriented interface for C++"
+HOMEPAGE="http://www.libgeodecomp.org/libflatarray.html"
+
+SLOT="0"
+LICENSE="Boost-1.0"
+IUSE="cuda doc"
+
+DEPEND="
+	doc? ( app-doc/doxygen )
+	cuda? ( dev-util/nvidia-cuda-toolkit )"
```

Comment:
This is most likely a BDEPEND. If it's only needed to execute a command during build-time, it's BDEPEND. If it has libraries and headers that are used / linked against while compiling, then it's a DEPEND. But seeing it's not required in runtime, I guess no linking happen either...

Reply:
The cuda headers are needed during compile time. Therefore, `depend` seems to be correct.

Reply:
And version v0.2.0 actually needs boost as well.